server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -485,6 +491,25 @@ private boolean execAdminCommand(String statement, long sessionId) throws Storag
     }
   }
 
+  private void execFlush(String statement) throws StorageGroupNotSetException {
+    String[] args = statement.split("\\s+");
+    if (args.length == 1) {
+      StorageEngine.getInstance().syncCloseAllProcessor();
+    } else if (args.length == 2){
+      String[] storageGroups = args[1].split(",\\s*");
```

Comment:
What do you want to match here? If the command is "FLUSH root.a.b,  root.b.c", the length if `args` is 3 which is invalid. So in your gramma, you don't allow space between `storageGroups`. So it's unnecessary to split `,\\s*`, just `,` is ok.

Reply:
fixed